### DISCUSSION TITLE
Баланс сват тим

### DISCUSSION THREAD

**Discussion by woenl** [octothorp-team/public-dobrograd#614] [Предложения]
Не кажится ли вам что сват через чур сильный, я понимаю они проходили спец подготовку. Но как они на себе носят полный бронежилет, П90, M4, XM, баллистический щит, аптечку, usp-s, bean bag, AWP и обычные вещи офицера. Где и как?

**Comment by 321ray** [comment]
Если оружие находится в хот-баре, не значит, что оно прямо сейчас на человеке. Взятие другого вида оружия обыгрывается через локер автомобиля, тоже самое с щитом. Обычные вещи офицера - на тактическом поясе, аптечка - на разгрузке, пистолет - в кобуре. Не понимаю смысл этого предложения, если это не вносит никаких новых механик или подобного, это всё равно было согласовано с ролевыми менеджерами, тебе лучше написать им =/

**Comment by Kyper1** [comment]
Не уверен на счет оружия, но вот живучесть у них жесткая, перестрелять одного почти невозможно.

**Comment by rasusibtisam** [comment]
Это же игровая условность насчёт всего того оружия у них в хот баре и конечно оно у них по рп не может быть на одном лице. Это как таран у копов.
